registrationform.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Registration Form</title>
+</head>
+
+<body>
+
+    <!-- Full Name Email Passord inputs. -->
+
+    <fieldset>
+        <legend>Basic User Information:</legend>
+
+        <label for="full-name"> Full name:</label>
+        <input type="text" id="full-name" name="full-name" value=""> <br>
+        <label for="email"> Email:</label>
+        <input type="text" id="email" name="email" value=""> <br>
+        <label for="password"> Password:</label>
+        <input type="password" id="password" name="password" value="">
+        <label for="password"> Confirm Password:</label>
+        <input type="password" id="confirm-password" name="confirm-password" value=""> <br>
+
+        <!-- Gender Input. -->
+        <p>Gender:
+            <input type="radio" id="male" name="gender" value="Male">
+              <label for="male">Male</label>
+              <input type="radio" id="female" name="gender" value="Female">
+              <label for="female">Female</label>
+              <input type="radio" id="other" name="gender" value="Other">
+              <label for="other">Other</label><br>
+        </p>
+
+        <!-- Birth day Input. -->
+
+        <label for="birthday">Birthday:</label>
+        <input type="date" id="birthday" name="birthday">
+
+    </fieldset>
+
+    <fieldset>
+        <legend>Adress Information:</legend>
+        <form action="/action_page.php">
+            <p>Street Address: <textarea name="message" rows="3" cols="20"> </textarea>
+            </p>
+            <p>Country: <select>
+                    <option value="USA"> USA</option>
+                    <option value="Moldova"> Moldova</option>
+                    <option value="UK"> UK</option>
+                    <option value="Romania"> Romania</option>
+                </select>
+                <br>
+
+                <label for="city"> City:</label>
+                <input type="text" id="city" name="city" value=""> <br>
+
+                <label for="postal-code"> Postal Code:</label>
+                <input type="text" id="postal-code" name="postal-code" value=""> <br>
+
+        </form>
+    </fieldset>
+
+    <fieldset>
+        <legend>Preferences:</legend>
+        <form action="/action_page.php">
+            <p> Preffered Contact Method:
+                <input type="checkbox" id="method1" name="method1" value="Email">
+                <label for="method1"> Email</label>
+                <input type="checkbox" id="method2" name="method2" value="Phone">
+                <label for="method2"> Phone</label>
+                <input type="checkbox" id="method3" name="method3" value="Postal Mail">
+                <label for="method3"> Postal Mail</label><br>
+            </p>
+
+            <p> Favorite Color:
+                <label for="favcolor"></label>
+                <input type="color" id="favcolor" name="favcolor">
+            </p>
+            <p>
```

Comment:
what is the reason to wrap a form element into a paragraph?

Reply:
fixed that too